config/samples/mysql_v1alpha1_mysqlcluster_backup_schedule_demo.yaml

```diff
@@ -11,11 +11,9 @@ spec:
   
   # if you want create mysqlcluster from S3, uncomment and fill the directory in S3 bucket below:
   # restoreFrom: 
-  BackupSchedule: "0 50 * * * *" 
+  backupSchedule: "0 50 * * * *" 
   mysqlOpts:
     image: percona/percona-server:5.7.34
```

Comment:
Get the name of the image directly from the CR

Reply:
it is a mysql cluster cr， backup cr image can fit nothing，because it has default value